lib/puppet_metadata/aio.rb

```diff
@@ -0,0 +1,42 @@
+module PuppetMetadata
+  class AIO 
+    BUILDS = { 
+      # RPM-based
+      'CentOS' => {
```

Comment:
I know I suggested this code earlier, but perhaps this is closer to the truth?
```suggestion
      'RedHat' => {
```

Reply:
I am not a user of RPM based platforms so can't tell which way is better, but as CentOS is a RedHat derivative it makes sense for me to be closer to the "root" for this hash.